projects/v2/src/app/app.module.ts

```diff
@@ -89,13 +63,31 @@ import { environment } from '../environments/environment';
       disabled: environment.production,
     }),
     NgxsResetPluginModule.forRoot(),
-    FormsModule,
-    ReactiveFormsModule,
-    MaterialModule,
-    NgxMatSelectSearchModule,
     OrderModule,
     FontAwesomeModule,
-    MarkdownModule.forRoot()
+    MarkdownModule.forRoot(),
+    CompareModule,
+    ControlsModule,
+    DebugLogsModule,
```

Comment:
These modules should not be a part of the `app.module`. The DebugLogs Module should be a part of the RootModule. Please make adjustments for the same. Only the component that is using the module should be imported. 

This will help you understand the hierarchy. Take a look here

![Screen Shot 2021-04-28 at 10 46 03 AM](https://user-images.githubusercontent.com/22999894/116423708-ee8a9480-a80e-11eb-97d4-018d980da98e.png)

Reply:
Done